AzureTable/Microsoft.DataTransfer.AzureTable/Source/AzureTableSourceAdapter.cs

```diff
@@ -28,7 +28,7 @@ public AzureTableSourceAdapter(IAzureTableSourceAdapterInstanceConfiguration con
 
             string connectionString = System.Text.RegularExpressions.Regex.Replace(
                 configuration.ConnectionString, @"(TableEndpoint=https://)(.*\.)(documents)(\.azure\.com)",
-                m => m.Groups[1].Value + m.Groups[2].Value + "table.cosmosdb" + m.Groups[4].Value);
+                m => m.Groups[1].Value + m.Groups[2].Value + "table.cosmosdb" + m.Groups[4].Value).TrimEnd('/');
```

Comment:
Why won't you just include it in the regex itself and ignore during the replacement? Why allocate another string?

Reply:
And in general, I think this should be marked as a workaround for a bug in the SDK. How does the trailing slash matter?

Reply:
yes, it is a workaround. I will mark it as such. the trailing slash I have observed would lead to an error when connecting to a tables endpoint, and return a 400 error, which would seem like a server error and in reality, it was a '/' character in the connection string.

Reply:
I chose TrimEnd() to keep it consistent with the source and sink changes I was making

Reply:
It's about code efficiency, which usually takes precedence. If you want consistency, then you should extract it into a method and reuse, rather than duplicate. That's the basic coding practice.